examples/neo-push/server/src/server.ts

```diff
@@ -19,6 +19,15 @@ if (config.NODE_ENV === "production") {
     });
 }
 
+if (config.NODE_ENV === "development"){
+    debug("Development serving graphql-playground");
+    const expressPlayground = require('graphql-playground-middleware-express').default
```

Comment:
Could we use the `import` syntax here to be consistent with the rest of the codebase?